man/resolvectl.xml

```diff
@@ -531,6 +531,15 @@ _443._tcp.fedoraproject.org IN TLSA 0 0 1 19400be5b7a31fb733917700789d2f0a2471c0
         -- Matching type: SHA-256
 </programlisting>
     </example>
+
+    <example>
+      <title>Resolve local services</title>
+
+      <programlisting>$ resolvectl --type=PTR _googlecast._tcp.local
+$ resolvectl --type=SRV somenamefromprevious._googlecast._tcp.local
+$ resolvectl --type=A hostnamefromprevious.local
```

Comment:
Hmm, "somenamefromprevious", why "googlecast"? Please use something that is not a commercial brand.

Reply:
I haven't found a suitable something yet. Can someone help add it?

Reply:
Any service name would work fine. Perhaps use DLNA? I know that Samba can broadcast over mDNS for service discovery.

But just saying

_serviceidentifier

is sufficient most likely.

Reply:
CephFS supports discovering some parts of itself through normal DNS service records, the same would work with mDNS if they patched their lookup code, for example: https://docs.ceph.com/en/latest/rados/configuration/mon-lookup-dns/ 
https://tracker.ceph.com/issues/50869#change-195470